website/docs/reference/dbt-classes.md

```diff
@@ -98,9 +98,14 @@ col.numeric_type('numeric', 12, 4) # numeric(12,4)
 
 ### Properties
 
-- **name**: Returns the name of the column
+- **char_size**: Returns the maximum size for character varying columns
+- **column**: Returns the name of the column
+- **data_type**: Returns the data type of the column (with size/precision/scale included)
+- **dtype**: Returns the data type of the column (without any size/precision/scale included)
+- **name**: Returns the name of the column (alias for column)
```

Comment:
Might help to include language about how this differs from `column` above

Reply:
hey @dbeatty10 can you help clarify the difference?

Reply:
from looking at the issue, `name` is  another way to access the same info -- alias for column. so it's included for convenience or consistency with other naming conventions.

what about the below @matthewshaver :

name: Returns the name of the column (identical to column, provided as an alias).